packages/web/components/transactions/slide-over-content.tsx

```diff
@@ -42,27 +66,34 @@ export const SlideOverContent = ({
             <div className="flex flex-col items-center justify-center gap-2 text-center">
               <div className="text-h5">Swapped</div>
               <div className="text-body1 text-osmoverse-300">
-                March 14, 2024, 13:01
+                {formattedDate}
               </div>
             </div>
           </div>
           <div className="flex flex-col rounded-2xl border border-osmoverse-700 p-2">
             <div className="flex justify-between p-2">
               <div className="flex gap-4">
-                <Image
-                  alt="OSMO"
-                  src="/tokens/generated/osmo.svg"
+                <FallbackImg
+                  alt={tokenIn.token.denom}
+                  src={tokenIn.token.currency.coinImageUrl}
+                  fallbacksrc="/icons/superfluid-osmo.svg"
```

Comment:
Is this for a missing token icon? I think we've been using question-mark.svg for this in the past.